src/notation/view/widgets/editstyle.ui

```diff
@@ -5476,41 +5476,35 @@
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_40">
-           <item row="1" column="2">
-            <spacer name="horizontalSpacer_20">
-             <property name="orientation">
-              <enum>Qt::Horizontal</enum>
-             </property>
-             <property name="sizeHint" stdset="0">
-              <size>
-               <width>0</width>
-               <height>0</height>
-              </size>
+           <item row="8" column="2">
+            <widget class="QToolButton" name="resetLedgerLineLength">
+             <property name="text">
+              <string>...</string>
```

Comment:
All of these will need to be marked untranslatable, and it's preferable to leave them empty instead of adding '...'. See Jojo's comment: `<string notr="true"/>`